src/compliance/privacy/ccpa.md

```diff
@@ -0,0 +1,43 @@
+---
+group: compliance
+title: California Consumer Privacy Act
+---
+
+The [California Consumer Protection Act][1]{: target="_blank"} (CCPA) expands the rights of consumers in California to determine how their personal information is collected, stored, and used, with an emphasis on protecting consumers from the unauthorized sale or exchange or their personal information. The CCPA was enacted in 2018 and is effective as of January 1, 2020.
+
+The CCPA grants the following new rights to consumers:
+
+-  **Right to know** the categories of personal information about them that is collected, used, shared, or sold in the past 12 months.
```

Comment:
was collected? "Past 12 months" implies past tense. Also, I think the sentence reads better without "about them".

Reply:
The "about them" was added by legal.   Fixed the tense. Thanks!